commit b23efe879bbab6e992172df0a101b576bc5b2ea6
Author: Armin Ronacher <armin.ronacher@active-4.com>
Date:   Wed Sep 4 23:12:50 2013 +0600

    Fixed a failing test on 3.3

diff --git a/werkzeug/testsuite/urls.py b/werkzeug/testsuite/urls.py
index c63c577..b333e1b 100644
--- a/werkzeug/testsuite/urls.py
+++ b/werkzeug/testsuite/urls.py
@@ -146,7 +146,7 @@ class URLsTestCase(WerkzeugTestCase):
                           'http://xn--f-1gaa.com:8080/bam/baz')
 
     def test_iri_safe_quoting(self):
-        uri = b'http://xn--f-1gaa.com/%2F%25?q=%C3%B6&x=%3D%25#%25'
+        uri = 'http://xn--f-1gaa.com/%2F%25?q=%C3%B6&x=%3D%25#%25'
         iri = u'http://föö.com/%2F%25?q=ö&x=%3D%25#%25'
         self.assert_strict_equal(urls.uri_to_iri(uri), iri)
         self.assert_strict_equal(urls.iri_to_uri(urls.uri_to_iri(uri)), uri)
